scripts/check.sh

```diff
@@ -421,6 +453,22 @@ main() {
           ;;
       esac
       ;;
+    "vmware_esxi")
+      (echo >&2 "[+] Enumerating vmware_esxi")
+      check_vagrant_path
```

Comment:
missing check_vagrant_env_plugin?

Reply:
Good eye, was thinking whether or not this would be required, I made env files the way they could be sourced via bash as well.

Reply:
Although vagrant-env is in the documentation defined as a requirement users usually use the ./goad.sh -t check to validate that they meet all the requirements.

Considering that users often don't read the documentation, it would be good to add the check for vagrant-env because otherwise the following error occurs:

```bash
PS /home/rebelinux/Projects/Development Projects/GOAD/GOAD> ./goad.sh -t install -l GOAD -p vmware_esxi -m local
[✓] Task: install
[✓] Lab: GOAD
[✓] Provider: vmware_esxi
[✓] Method: local
[✓] folder ad/GOAD/providers/vmware_esxi found
[✓] Launch installation for: GOAD / vmware_esxi / local
[✓] launch vagrant

/home/rebelinux/.vagrant.d/gems/3.0.6/gems/vagrant-vmware-esxi-2.5.5/lib/vagrant-vmware-esxi/action/esxi_password.rb:115:in `set_esxi_password': undefined method `gsub' for nil:NilClass (NoMethodError)
        from /home/rebelinux/.vagrant.d/gems/3.0.6/gems/vagrant-vmware-esxi-2.5.5/lib/vagrant-vmware-esxi/action/esxi_password.rb:17:in `call'
        from /opt/vagrant/embedded/gems/gems/vagrant-2.4.1/lib/vagrant/action/warden.rb:38:in `call'
        from /opt/vagrant/embedded/gems/gems/vagrant-2.4.1/lib/vagrant/action/builder.rb:183:in `call'
        from /opt/vagrant/embedded/gems/gems/vagrant-2.4.1/lib/vagrant/action/runner.rb:104:in `block in run'
        from /opt/vagrant/embedded/gems/gems/vagrant-2.4.1/lib/vagrant/util/busy.rb:22:in `busy'
        from /opt/vagrant/embedded/gems/gems/vagrant-2.4.1/lib/vagrant/action/runner.rb:104:in `run'
        from /opt/vagrant/embedded/gems/gems/vagrant-2.4.1/lib/vagrant/machine.rb:247:in `action_raw'
        from /opt/vagrant/embedded/gems/gems/vagrant-2.4.1/lib/vagrant/machine.rb:216:in `block in action'
        from /opt/vagrant/embedded/gems/gems/vagrant-2.4.1/lib/vagrant/environment.rb:649:in `lock'
        from /opt/vagrant/embedded/gems/gems/vagrant-2.4.1/lib/vagrant/machine.rb:202:in `call'
        from /opt/vagrant/embedded/gems/gems/vagrant-2.4.1/lib/vagrant/machine.rb:202:in `action'
        from /home/rebelinux/.vagrant.d/gems/3.0.6/gems/vagrant-vmware-esxi-2.5.5/lib/vagrant-vmware-esxi/provider.rb:24:in `state'
        from /opt/vagrant/embedded/gems/gems/vagrant-2.4.1/lib/vagrant/machine.rb:544:in `state'
        from /opt/vagrant/embedded/gems/gems/vagrant-2.4.1/lib/vagrant/machine.rb:159:in `initialize'
        from /opt/vagrant/embedded/gems/gems/vagrant-2.4.1/lib/vagrant/vagrantfile.rb:86:in `new'
        from /opt/vagrant/embedded/gems/gems/vagrant-2.4.1/lib/vagrant/vagrantfile.rb:86:in `machine'
        from /opt/vagrant/embedded/gems/gems/vagrant-2.4.1/lib/vagrant/environment.rb:758:in `machine'
        from /opt/vagrant/embedded/gems/gems/vagrant-2.4.1/lib/vagrant/plugin/v2/command.rb:184:in `block in with_target_vms'
        from /opt/vagrant/embedded/gems/gems/vagrant-2.4.1/lib/vagrant/plugin/v2/command.rb:208:in `block in with_target_vms'
        from /opt/vagrant/embedded/gems/gems/vagrant-2.4.1/lib/vagrant/plugin/v2/command.rb:190:in `each'
        from /opt/vagrant/embedded/gems/gems/vagrant-2.4.1/lib/vagrant/plugin/v2/command.rb:190:in `with_target_vms'
        from /opt/vagrant/embedded/gems/gems/vagrant-2.4.1/plugins/commands/up/command.rb:90:in `execute'
        from /opt/vagrant/embedded/gems/gems/vagrant-2.4.1/lib/vagrant/cli.rb:67:in `execute'
        from /opt/vagrant/embedded/gems/gems/vagrant-2.4.1/lib/vagrant/environment.rb:319:in `cli'
        from /opt/vagrant/embedded/gems/gems/vagrant-2.4.1/bin/vagrant:248:in `<main>'
[!] vagrant finish with error abort
```